src/dc_chat.rs

```diff
@@ -1854,78 +1853,62 @@ pub unsafe fn dc_set_chat_name(
     success
 }
 
-// TODO should return bool /rtn
-#[allow(non_snake_case)]
 pub unsafe fn dc_set_chat_profile_image(
     context: &Context,
     chat_id: uint32_t,
-    new_image: *const libc::c_char,
-) -> libc::c_int {
-    let mut OK_TO_CONTINUE = true;
-    let mut success: libc::c_int = 0i32;
+    new_image: impl AsRef<str>,
```

Comment:
not sure it's pertinent to this PR but why is an image represented as a str? isn't there a better one like maybe Vec<u8> or so?

Reply:
It's not an image but a path to an image file.